submissions/reactjs/assignment2.MD

```diff
@@ -39,7 +39,7 @@
 | Vraj Mehta                     | 2303031087034        |                   |             |
 | Alisha Kaur                    | 2203031080202        |                   |             |
 | Pranav Faladu                  | 2303031087022        |                   |             |
-| Bharvi Valand                  | 2303031087002        |                   |             |
+| Bharvi Valand                  | 2303031087002        |[Assignment32](https://github.com/bharvivaland/ReactAssignments/blob/main/src/assets/Greet.jsx)|[Github](https://github.com/bharvivaland/ReactAssignments.git)|
```

Comment:
Please follow the repository structure mentioned in the problem statement document